validator/validator-main.protoascii

```diff
@@ -1494,7 +1494,29 @@ tags: {
     value_regex: "[0-9]*"
   }
 }
-# 4.4.8 The dl element
+# 4.11.1 (HTML5.3) The details element
```

Comment:
It seems with 5.3 they've renumbered everything so this doesn't really follow our existing numbering. I'd still recommend putting it just before `4.11 Scripting` below and include a `4.11 Interactive elements` before it.

Reply:
Done in 7255acd0